app/main.py

```diff
@@ -1,25 +1,87 @@
 from abc import ABC
+from typing import Any
 
 
 class IntegerRange:
-    pass
+
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: type, name: str) -> None:
+        self.protected_name = f"_{name}"
+
+    def __get__(self, instance: Any, owner: type) -> None:
```

Comment:
The return type of the `__get__` method should be `Any` instead of `None`. This method is expected to return the value of the attribute it manages.